src/uct/ib/mlx5/gdaki/configure.m4

```diff
@@ -40,8 +42,8 @@ AS_IF([test x$with_doca_gpunetio != xno], [
 
 save_CPPFLAGS="$CPPFLAGS"
 save_LDFLAGS="$LDFLAGS"
-CPPFLAGS="$CPPFLAGS $GPUNETIO_CFLAGS"
-LDFLAGS="$LDFLAGS $GPUNETIO_LDFLAGS"
+CPPFLAGS="$CPPFLAGS $CUDA_CFLAGS $GPUNETIO_CFLAGS"
+LDFLAGS="$LDFLAGS $CUDA_LDFLAGS $CUDA_LIBS $CUDART_LIBS $GPUNETIO_LDFLAGS"
```

Comment:
we want to avoid cudart and use only driver API, is it needed for testing of DOCA functions?
also i'd expect DOCA to be self contained

Reply:
gpunetio depends on -lcudart
maybe add -rpath-link to CUDA_LDFLAGS?
then it will find w/o CUDART_LIBS